src/verify-request.ts

```diff
@@ -0,0 +1,21 @@
+import { verifyRequestSignature } from '@slack/events-api';
+
+export const verifyRequest = (headers: { [key: string]: any }, body: any) => {
+  try {
+    const requestSignature = headers['X-Slack-Signature'] as string;
+    const requestTimestamp = headers['X-Slack-Request-Timestamp'] as string;
+    if (!requestSignature || !requestTimestamp) throw 'Slack request signing verification failed';
+
+    const params = {
+      signingSecret: process.env.SLACK_SIGNING_SECRET || '',
```

Comment:
The other files use the name `SLACK_SIGNING_KEY`, shouldn't this be the same?

Reply:
Eek, yep.